piet-common/src/cairo_back.rs

```diff
@@ -121,4 +126,19 @@ impl<'a> BitmapTarget<'a> {
         }
         Ok(raw_data)
     }
+
+    pub fn save_to_file(self, path: &str) -> Result<(), piet::Error> {
```

Comment:
for `path`, I think the best type for the signature would be `impl AsRef<Path>`; this allows a larger number of acceptable types to be used here, and is the same type that `File::create` takes.